src/gui/CMakeLists.txt

```diff
@@ -189,6 +189,8 @@ IF( WIN32 )
     ENDIF()
 ENDIF()
 
+include(libcloudproviders/libcloudproviders.cmake)
```

Comment:
This is rather unusual.
*Usually* you would probably put it in CMakeLists.txt file, add it as a subdirectory and define a static target in there (with include dirs and linked libraries) against which you then can link the application. 
The way you did it works as well of course.